### ISSUE TITLE
Visual errors in Graph View

### ISSUE THREAD

**Issue by WinnerWind** [KosmosisDire/obsidian-webpage-export#319]
**Describe the bug**
When hosting the server, the graph view has a tendency to glitch out and lose its text colour. 
Also, when using the Gummy Theme, the graph view sometimes becomes just lines, even though the graph view in Obsidian is just fine. This is sporadic and does not seem to show a pattern as to why. Sometimes, the nodes do appear, but a refresh removes the nodes.
When using Material Flat, the text colour on the light theme is white by default, and when setting it to dark, it stays white, but when changing it bac to the light theme, the text colour is as expected (black)

**Operating System**
Windows 10. Using HTTP-SERVER to host.

**To Reproduce**
### Using the default theme + Material Flat Theme
Host the server using any web hosting service (I used HTTP server because it's local) 
See the graph view, which appears to be in white.
CTRL + F5 does not fix the issue.

### Gummy 
Host the server using any web hosting server
See the graph view
If it has nodes, refresh the page
The nodes dissapear
CTRL + F5 does not fix the issue

**Videos**
### General Refresh test on Default Theme (Note how text becomes white at 0:12)
https://youtu.be/YNQEBNqz7ZE
### Forced Refresh test on Gummy Theme (CTRL + F5). Note how the nodes never appeared
https://youtu.be/ZMubEIuEZ3U
### General Refresh Test on Material Flat
https://youtu.be/H6ZTCg_vQtM
### General Refresh test after graph view is broken (Gummy)
https://youtu.be/zggKqvsoAN4

### Gummy Theme
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/71366717/4dcad06f-faf9-4b27-a799-5653e6e2745c)
### Default Theme
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/71366717/f388915c-31c5-43de-8ec0-dddb1f4abed2)
(Material Flat shows same behavior)

**Comment by KosmosisDire** [comment]
This appears to be issue #124 which I thought I had fixed. I will try and reproduce this again and fix it

**Comment by KosmosisDire** [comment]
Actually, what version of the plugin are you using? Is it possible you are suing an old version (or have not done a full reexport since an old version)?

**Comment by KosmosisDire** [comment]
1.9.6 is not a valid version of the plugin. The latest version is 1.7.3, but it may not be showing updated because you may be using an old installer for obsidian. Can you run the debug info command in your vault and include that info here? It would be very helpful.

**Comment by WinnerWind** [comment]
Hi there! I am so sorry! 
I am indeed on 1.7.3. That version number was for another, similar plugin.
Heres my debug info
```
SYSTEM INFO:
	Obsidian version: v1.5.3
	Installer version: v1.5.3
	Operating system: Windows 10 Pro 10.0.19045
	Login status: logged in
	Catalyst license: none
	Insider build toggle: off
	Live preview: on
	Base theme: adapt to system
	Community theme: gummy v1.0.0.1
	Snippets enabled: 1
	Restricted mode: off
	Plugins installed: 5
	Plugins enabled: 3
		1: Webpage HTML Export v1.7.3
		2: Style Settings v1.0.7
		3: Badges v1.1.0

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Comment by KosmosisDire** [comment]
Thanks, I am not able to reproduce this on the default theme in the beta version. However, I am able to reproduce the problem using the gummy theme. I will look into fixing that

**Comment by KosmosisDire** [comment]
This should be fixed in 1.8.0. Let me know if it still does not work as expected and reopen the issue. If you find any new issues please submit a new issue :)
Thanks!
